Gemfile.lock

```diff
@@ -175,6 +177,23 @@ GEM
     reline (0.3.1)
       io-console (~> 0.5)
     rexml (3.2.5)
+    rspec-core (3.11.0)
+      rspec-support (~> 3.11.0)
+    rspec-expectations (3.11.0)
+      diff-lcs (>= 1.2.0, < 2.0)
+      rspec-support (~> 3.11.0)
+    rspec-mocks (3.11.1)
```

Comment:
- [ ] Please add a test for validation in the user test I see that you just created an empty one 
![image](https://user-images.githubusercontent.com/74744787/185722892-ccc48b61-d769-4f98-a2ce-bd501bac19d8.png)
it should look like this 
![image](https://user-images.githubusercontent.com/74744787/185722902-7fbb2e71-136e-4afc-899e-52aa0984b2fa.png)

Reply:
Sir @MeqdamAlqudah, I believe that what you suggest is optional for this project and not required.

Reply:
Hello, @MeqdamAlqudah  I believe that this is optional, in the project requirements there is only: Add unit specs for all of your models' methods and validations.